### PR TITLE
`treehouses led wazazi` (fixes #1462)

### LINKED ISSUE #1462: `treehouses led <festivity>`?

### PR DESCRIPTION
**Pull request by sarahrams**
fixes issue #1462 

Description 
Creatiing an LED sub-command for a festivity

### CODE REVIEW

File: modules/led.sh [hunk 1]
```diff
@@ -100,7 +100,18 @@ function led {
       echo "Green LED: on 2 sec; off 0.5 sec"
       echo "Red LED: on 2 sec; off 0.5 sec"
       eid > "$LOGFILE"
-      ;;
+      ;;
```
Comment (law-lin) [inline modules/led.sh:103]: There seems to be a trailing space on line 103

File: modules/led.sh [hunk 2]
```diff
@@ -100,7 +100,18 @@ function led {
...
+      checkroot
+      echo "leds are set to wazazi mode."
+      echo "Look at your RPi leds, both leds will be in this pattern..."
+      echo "Both LED: 1 sec off"
+      echo "Green LED: flash 6 times, on 2 sec"
+      echo "Red LED: 0.5 on, 0.25 off, 3 times"
+      echo "Green LED: 0.5 on, 0.25 off, 3 times"
+      echo "Both LED: 1 sec off"
+      newyear > "$LOGFILE"
+      ;;
```
Comment (law-lin) [inline modules/led.sh:114]: Trailing space

File: modules/led.sh [hunk 3]
```diff
@@ -776,6 +816,9 @@ function led_help {
   echo "  $BASENAME led eid"
   echo "     This will set the mode of the led to eid"
   echo
+  echo "  $BASENAME led wazazi"
+  echo "      This will set the mode of the led to wazazi"
+  echo 
   echo "  $BASENAME led onam"
```
Comment (law-lin) [inline modules/led.sh:822]: Trailing space

File: modules/led.sh [hunk 4]
```diff
@@ -100,7 +100,18 @@ function led {
...
       echo "Red LED: on 2 sec; off 0.5 sec"
       eid > "$LOGFILE"
-      ;;
+      ;;      
+    wazazi)
+      checkroot
+      echo "leds are set to wazazi mode."
+      echo "Look at your RPi leds, both leds will be in this pattern..."
+      echo "Both LED: 1 sec off"
+      echo "Green LED: flash 6 times, on 2 sec"
```
Comment (law-lin) [inline modules/led.sh:109]: Not too sure about this pattern description, the LED lights seem to flash 6 times alternating between green and red

File: README.md [hunk 5]
```diff
@@ -70,7 +70,7 @@ sshtunnel [add|remove|list|check|key]     helps adding sshtunnels
 led [green|red] [mode]                    sets the led mode
     [dance|thanksgiving|christmas]
     [newyear|lunarnewyear|valentine]
-    [carnival|stpatricks|onam]
+    [wazazi|carnival|stpatricks|onam]
     [heavymetal|easter|eid|kecak|random]
```
Comment (dogi) [inline README.md:74]: should be the same as in the help

**Comment by jayala-29** [comment]
@dogi should this PR still be open?

File: tests/led.bats [hunk 6]
```diff
@@ -56,6 +56,11 @@ load test-helper
   assert_success && assert_output -p 'heartbeat'
 }
 
+@test "$clinom led wazazi" {
+  run "${clicmd}" led wazazi
+  assert_success && assert_output -p 'heartbeat'
```
Comment (darthnoward) [inline tests/led.bats:61]:
```
root@treehouses:~/test/cli# bats tests/led.bats
...
 ✗  led wazazi
   (from function `assert_output' in file /usr/lib/node_modules/bats-assert/src/assert.bash, line 247,
    in test file tests/led.bats, line 61)
     `assert_success && assert_output -p 'heartbeat'' failed

   -- output does not contain substring --
   substring (1 lines):
     heartbeat
   output (9 lines):
     leds are set to wazazi mode.
     Look at your RPi leds, both leds will be in this pattern...
     Both LED: 1 sec off
     Both LED: flash 6 times, on 2 sec
     Red LED: 0.5 on, 0.25 off, 3 times
     Green LED: 0.5 on, 0.25 off, 3 times
     Both LED: 1 sec off
     red led: default-on
```
here should be 
```
assert_success && assert_output -p 'default-on'
```

**Pull request closed without merging**